packages/server/test/unit/gui/events_spec.js

```diff
@@ -669,68 +676,23 @@ describe('lib/gui/events', () => {
       })
 
       it('catches errors', function () {
+        this.create = sinon.stub(OpenProject.prototype, 'create').resolves()
         const err = new Error('foo')
 
-        this.open.rejects(err)
+        this.create.rejects(err)
 
         return this.handleEvent('open:project', '/_test-output/path/to/project-e2e')
         .then((assert) => {
           return assert.sendErrCalledWith(err)
         })
       })
 
-      it('sends \'focus:tests\' onFocusTests', function () {
```

Comment:
I think we lose some unit test coverage here. These tests relied on the fact that `open:project` calls `openProject#create` which previously would call `openProject#open`, which is no longer the case, so I'm not sure these particular unit tests are relevant anymore.

That said, I am not sure where in the suite this behavior *is* covered.